Userland/Services/WindowServer/SystemEffects.h

```diff
@@ -81,18 +81,28 @@ class SystemEffects {
     ~SystemEffects() = default;
 
     Vector<bool>& effects() { return m_effects; }
-    bool animate_menus() const { return m_effects[AnimateMenus]; }
-    bool flash_menus() const { return m_effects[FlashMenus]; }
-    bool animate_windows() const { return m_effects[AnimateWindows]; }
-    bool smooth_scrolling() const { return m_effects[SmoothScrolling]; }
-
-    bool tab_accents() const { return m_effects[TabAccents]; }
-    bool splitter_knurls() const { return m_effects[SplitterKnurls]; }
-    bool tooltips() const { return m_effects[Tooltips]; }
-
-    bool menu_shadow() const { return m_effects[MenuShadow]; }
-    bool window_shadow() const { return m_effects[WindowShadow]; }
-    bool tooltip_shadow() const { return m_effects[TooltipShadow]; }
+    bool set_effect(Effects effect, bool value)
+    {
+        VERIFY(static_cast<size_t>(effect) < static_cast<size_t>(Effects::__Count));
+        auto& effect_value = m_effects[static_cast<size_t>(effect)];
```

Comment:
In the VERIFY, you shouldn't need to cast, enum classes are comparable:
```
VERIFY(effect < Effects::__Count);
```

If someone does something sneaky like `set_effect(static_cast<Effects>(123456789))`, it'll still be caught.